src/components/RequirePermissions/index.tsx

```diff
@@ -0,0 +1,14 @@
+import { hasComponent } from '@lfai/egeria-js-commons';
+import { AccessDenied } from '../UserPermissions/AccessDenied';
+
+export function RequirePermissions(props:any) {
+  if (hasComponent(props.component)) {
+    return (<>{props.element}</>);
+  } else {
+    if (props.showAccessDenied) {
+      return (<AccessDenied component={props.component}/>);
+    } else {
+      return (<></>);
+    }
+  }
+}
```

Comment:
don't forget about the '*' for super admin rights

Reply:
i think it applies here https://github.com/odpi/egeria-js-commons/pull/88/files#diff-cdf05ce4e1beabfb807cea0f0ca01f483406aeb8353a109cd003907482175a1dR3

Reply:
resolved by "@lfai/egeria-js-commons": "^4.2.16"